codes/quantum/qubits/quantum_reed_muller.yml

```diff
@@ -10,7 +10,12 @@ logical: qubits
 name: 'Quantum Reed-Muller code'
 introduced: '\cite{arXiv:quant-ph/9608026}'
 
-description: 'A CSS code formed from a classical Reed-Muller code in which polynomials over finite fields encode data. This is done by transforming these polynomials into the stabilizer generator matrices.'
+description: 'A CSS code formed from a classical Reed-Muller code in which polynomials over finite fields encode data. This is done by transforming these polynomials into the stabilizer generator matrices. 
+The pure stabilizer code \([[q^m,k(v_2)-k(v_1),min{d(v_1^{\perp}),d(v_2)}]]_{q}\) with parameters \(k(v) = \sum_{j=0}^{m}(-1)^{j}\dbinom{m}{j}\dbinom{m+v-jq}{v-jq}\) and 
```

Comment:
There's still q's here; I thought this was defined for qubits (q=2) only. Where did you get these eqns from?